django_prices/forms.py

```diff
@@ -1,49 +1,69 @@
 import itertools
+from typing import List, Optional, Tuple
 
+from babel.numbers import get_currency_symbol
 from django import forms
+from django.core.validators import ValidationError
 from prices import Money
 
-from .validators import (
-    MaxMoneyValidator,
-    MinMoneyValidator,
-    MoneyPrecisionValidator,
-    ValidationError,
-)
+from .validators import MaxMoneyValidator, MinMoneyValidator, MoneyPrecisionValidator
 from .widgets import FixedCurrencyMoneyInput, MoneyInput
+from .templatetags.prices_i18n import get_locale_data
```

Comment:
This import should come before `.validators`.

Reply:
Please move this utility function elsewhere. It does not make sense for forms to import template tags.